cmd/kube-score/main.go

```diff
@@ -99,6 +98,7 @@ func scoreFiles(binName string, args []string) error {
 	exitOneOnWarning := fs.Bool("exit-one-on-warning", false, "Exit with code 1 in case of warnings")
 	ignoreContainerCpuLimit := fs.Bool("ignore-container-cpu-limit", false, "Disables the requirement of setting a container CPU limit")
 	ignoreContainerMemoryLimit := fs.Bool("ignore-container-memory-limit", false, "Disables the requirement of setting a container memory limit")
+	ignoreContainerStorageEphemeralLimit := fs.Bool("ignore-container-ephemeral-storage-limit", false, "Disables the requirement of setting a container ephemeral-storage limit")
```

Comment:
kube-score has a better mechanism for ignoring checks these days, via the `--ignore-test` flag and `kube-score/ignore` annotation.

Could you use that mechanism instead? This means that you will have to add this feature as a new check, instead of adding it to `Container Resources`, but I think that's a good trade-off.